co2eq/purchase/index.js

```diff
@@ -55,6 +55,7 @@ function indexNodeChildren(branch, i = 1) {
     }
```

Comment:
The model version should be bumped so we recalculate all purchases now, right? 🤔